website/docs/r/imagebuilder_container_recipe.html.markdown

```diff
@@ -65,6 +65,7 @@ The following attributes are optional:
 * `dockerfile_template_uri` - (Optional) The Amazon S3 URI for the Dockerfile that will be used to build the container image.
 * `instance_configuration` - (Optional) Configuration block used to configure an instance for building and testing container images. Detailed below.
 * `kms_key_id` - (Optional) The KMS key used to encrypt the container image.
+* `platform_override` - (Optional) The platform override is used when referencing a ECR image instead of an Image Builder image. Valid values are `Linux` and `Windows`.
```

Comment:
I'd normally defer to the [API docs](https://pkg.go.dev/github.com/aws/aws-sdk-go-v2/service/imagebuilder#CreateContainerRecipeInput) here, but I actually think a combination of yours + theirs might work quite nicely, though I admit this is very much in the hair-splitting category:

```suggestion
* `platform_override` - (Optional) This specifies the operating system platform and is required when you use a custom base image in `parent_image` instead of an Image Builder Image. Valid values are `Linux` and `Windows`.
```